.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+configs/.env
```

Comment:
Does this works. I can see the env file

Reply:
yes! you have to rename `configs/.env.env` file to .env and add your mongo uri in order to work
after renaming it will be `configs/.env` and git will ignore this file